internal/run.bzl

```diff
@@ -36,24 +36,25 @@ def _run_one(ctx, input_css, input_map, output_css, output_map):
         "--binDir=%s" % ctx.bin_dir.path,
         "--cssFile=%s" % input_css.path,
         "--outCssFile=%s" % output_css.path,
-        "--outCssMapFile=%s" % output_map.path,
         "--data=%s" % ','.join([f.path for f in data.to_list()]),
         "--additionalOutputs=%s" % ','.join([f.path for f in additional_outputs])
     ]
-    if input_map:
-        args.append("--cssMapFile=%s" % input_map.path)
+    if input_map: args.append("--cssMapFile=%s" % input_map.path)
```

Comment:
Nit: Break into newlines? Buildifier will do this

Reply:
Done.